src/main/java/uk/gov/digital/ho/hocs/audit/export/caseworkclient/CaseworkClient.java

```diff
@@ -58,4 +55,16 @@ public CaseworkClient(RestHelper restHelper,
             return Collections.emptySet();
         }
     }
+
+    public GetCaseReferenceResponse getCaseReference(String uuid) {
```

Comment:
We should add spring caching to this method too, check InfoClient for ways it could be done.  This would mean from this service there would be no outgoing calls on the second extract that we run. (I'd still keep the caching on the other end). We know that caseRef will never change for case UUID so caching indefinitely here is beneficial.